config/locales/errors/en.yml

```diff
@@ -20,8 +20,14 @@ en:
     doc_auth:
       acuant_throttle: To prevent abuse we limit the amount of times you can attempt
         to validate a document.  Please try again later.
-      general_error: Your state issued ID could not be verified.  Please try uploading
-        different images starting with the front of your state issued ID.
+      general_error_html: Your state issued ID could not be verified.  Please try
```

Comment:
I'd suggest breaking this up into two different pieces, one with the error and one with the list of suggestions. If possible, could we make the list of suggestions an array with no HTML in the config files? It might require re-working some other things but would make it easier to manage these translations and style them in the view

Reply:
Implemented. Please see screenshot in the description. Thank you @donjo !